internal/cli/mongocli/alerts/settings/list_test.go

```diff
@@ -25,7 +25,6 @@ import (
 	"github.com/mongodb/mongodb-atlas-cli/mongocli/v2/internal/mocks"
 	"github.com/mongodb/mongodb-atlas-cli/mongocli/v2/internal/pointer"
 	"github.com/stretchr/testify/assert"
-	"go.mongodb.org/atlas/mongodbatlas"
```

Comment:
does it help to do `go mod tidy` if not used anymore?

Reply:
we have a test check to make sure is needed but in this case mocks for some reason still use the atlas version (ops manager defines an alias) and `mongocli auth login` still needs this lib, I think the new team that will take over mongocli will be cleanign some of that and why this PR